src/Core/src/Core/IView.cs

```diff
@@ -47,7 +47,7 @@ public interface IView : IElement, ITransform
 		/// <summary>
 		/// Gets a value indicating whether this View is enabled in the user interface. 
 		/// </summary>
-		bool IsEnabled { get; }
+		bool IsEnabled { get; set; }
```

Comment:
This should not have a setter i don't think. The tests can have it, no need to add it to the interface as well.

Reply:
Right, changed.